tasks/main.yml

```diff
@@ -21,6 +21,8 @@
 - debug: 'msg="[DEPRECATION NOTICE] Using `datadog_process_checks` is deprecated, use `process` under `datadog_checks` instead"'
   when: datadog_process_checks is defined
 
+- service: name=dd-trace-agent state=removed
```

Comment:
I this should be `yum` module :)
Also let's put this into the `pkg-redhat.yml`.